src/Buttons.js

```diff
@@ -0,0 +1,25 @@
+import React, {useState} from "react";
+import {homeScore} from "./App.js"
+
+const Buttons = (props) => { 
+// function Buttons(props) {
+    console.log(props.homeScore)
+    // const [homeScore, setHomeScore] = useState(0)
+    // const [awayScore, setAwayScore] = useState(0)
```

Comment:
Great job recognizing we need to have the state within App, so we can pass it down to other components that need it.